site/app/views/grading/ElectronicGraderView.php

```diff
@@ -50,7 +50,7 @@ public function statusPage(
     ) {
 
         $peer = false;
-        if ($gradeable->isPeerGrading() && $this->core->getUser()->getGroup() == User::GROUP_STUDENT) {
+        if ($gradeable->isPeerGrading()) {//&& $this->core->getUser()->getGroup() == User::GROUP_STUDENT) {
             $peer = true;
         }
```

Comment:
This can just be simplified to `$peer = $gradeable->isPeerGrading()` instead of having the if structure.